readme.md

```diff
@@ -4,7 +4,8 @@ It prompts the user to answer some questions to create configuration files autom
 ## Warning
 If you have already installed `KoBoToolbox` with `kobo-docker` prior March 2019,
 databases are not compatible and and docker images (`PostgreSQL`, `MongoDB`) are not the same.  
-**`KoBoInstall` won't be able to start the app.**
+**`KoBoInstall` won't be able to start the app.**  
+Follow [those steps](https://github.com/kobotoolbox/kobo-docker#upgrading-from-an-old-version-of-kobo-docker-before-to-march-2019) to upgrade.
 
```

Comment:
Thank you. 
It's a good idea to tell users how to upgrade `kobo-docker in `kobo-install` repo.
Do you think, we should insist on the fact, users should not try to run `kobo-install` before upgrading `kobo-docker` to correct versions?

Reply:
Honestly, I don't know what to answer here.

I did installed from kobo-install, at first, So when it was time to upgrade I had no idea I should check on kobo-docker repo.

Then I've just started a new repo from scratch, using kobo-install. So I didn't try to actually follow the upgrade steps.

Reply:
The warning seems clear enough to me. But I'm an IT guy, I don't know how your regular users react.